de.nagstamon.nagstamon.yml

```
@@ -0,0 +1,114 @@
+app-id: de.nagstamon.nagstamon
+runtime: org.kde.Platform
+runtime-version: '6.7'
+sdk: org.kde.Sdk
+sdk-extensions:
+  - org.freedesktop.Sdk.Extension.rust-stable
+base: com.riverbankcomputing.PyQt.BaseApp
+base-version: '6.7'
```

```suggestion
runtime-version: '6.6'
sdk: org.kde.Sdk
sdk-extensions:
  - org.freedesktop.Sdk.Extension.rust-stable
base: com.riverbankcomputing.PyQt.BaseApp
base-version: '6.6'
```

My bad I realised that the PyQt base app isn't available for that runtime. And it will probably be a while.